### PR TITLE
docs: fix subscription example and add output-location note  

### LINKED ISSUE #502: SWA CLI documentation improvement
Not really a bug, I chose the bug option as it was the one most fitting.

**Describe the bug**
Using `swa deploy --app-location ./packages/frontend --output-location ./packages/frontend/dist` gives the error `The folder "/home/runner/work/***/***/packages/frontend/packages/frontend/dist" is not found. Exit`. 

This shows that the output location is relative to the app location. This is not noted (or at least, wasn't clear) in [the documentation](https://azure.github.io/static-web-apps-cli/docs/cli/swa-deploy/#options).

**To Reproduce**

1. Go to the root of your monorepo project
2. Type in the command `swa deploy --app-location ./packages/frontend --output-location ./packages/frontend/dist`
3. See error

**Expected behavior**
After reading the documentation, I expected that the output location would not be relative to the app location. In my opinion, it makes sense that it is, the documentation just needs to be updated to be more clear.

**Desktop (please complete the following information):**
- OS: macOS
- Version 12.2.1

### LINKED ISSUE #504: error: unknown option '--subscription'
Before filing this issue, please ensure you're using the latest CLI by running `swa --version` and comparing to the latest version on [npm](https://www.npmjs.com/package/@azure/static-web-apps-cli).

**Are you accessing the CLI from the default port `:4280` ?**

N/A

> Make sure you are accessing the URL printed in the console when running `swa start`!

**ℹ️ NOTE: Make sure to enable debug logs when running any `swa` commands using `--verbose=silly`**

**Describe the bug**
When I run `swa login --subscription <subscription-id>` I get this error: `error: unknown option '--subscription'`, but `swa login --help` says it should be `subscription`

**To Reproduce**

1. Run `swa login --help` and see how the help says the parameter is `--subscription`
2. Run `swa login --subscription <subscription-id>`
3. See how it says  `error: unknown option '--subscription'`

**Expected behavior**
Either the help should say its `subscription-id` or the option should be changed from `subscription-id` to `subscription`.

**Desktop (please complete the following information):**

- OS: Windows
- Version: 10

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Reshmi-Sriram**
Fixes #504 and Fixes #502

### CODE REVIEW

File: docs/www/docs/cli/swa-start.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Here are the options you can use with `swa login`:
 
 - `-a, --app-location <path>`: the folder containing the source code of the front-end application (default: ".")
 - `-i, --api-location <path>`: the folder containing the source code of the API application
-- `-O, --output-location <path>`: the folder containing the built source of the front-end application (default: ".")
+- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to the app-location (default: ".")
```
Comment (manekinekko) [inline docs/www/docs/cli/swa-start.md:109]:
```suggestion
- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to --app-location (default: ".")
```

File: docs/www/docs/cli/swa-deploy.md [hunk 2]
```diff
@@ -152,7 +156,7 @@ Here are the options you can use with `swa deploy`:
 
 - `-a, --app-location <path>`: the folder containing the source code of the front-end application (default: "`.`")
 - `-i, --api-location <path>`: the folder containing the source code of the API application
-- `-O, --output-location <path>`: the folder containing the built source of the front-end application (default: "`.`")
+- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to the app-location (default: "`.`")
```
Comment (manekinekko) [inline docs/www/docs/cli/swa-deploy.md:159]:
```suggestion
- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to --app-location (default: "`.`")
```

File: docs/www/docs/cli/swa-build.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Here are the options you can use with `swa build`:
 
 - `-a, --app-location <path>`: the folder containing the source code of the front-end application (default: ".")
 - `-i, --api-location <path>`: the folder containing the source code of the API application
-- `-O, --output-location <path>`: the folder containing the built source of the front-end application (default: ".")
+- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to the app-location (default: ".")
```
Comment (manekinekko) [inline docs/www/docs/cli/swa-build.md:31]:
```suggestion
- `-O, --output-location <path>`: the folder containing the built source of the front-end application. The path is relative to --app-location (default: ".")
```

**Review by manekinekko — CHANGES_REQUESTED** [review]

**Comment by Reshmi-Sriram** [comment]
Made changes. Pls review @manekinekko @sgollapudi77

**Review by sgollapudi77 — APPROVED** [review]

**Pull request merged**